code/controllers/subsystem/ai_move_test.dm

```diff
@@ -0,0 +1,44 @@
+//Potential implementation of a tile by tile movement via subsystem
+
+SUBSYSTEM_DEF(ai_movement)
```

Comment:
this is going to overwrite completely the old file, needs to be resolved at some point

Reply:
Commented out for now in case this ever needs to be reused; lots of commented out code in this pr